--- a/build/make/Makefile	2013-07-19 17:38:27 -0400
+++ b/build/make/Makefile	2013-07-19 17:57:31 -0400
@@ -162,9 +162,8 @@
 # keep them.
 %.a: %_g.a
 	$(if $(quiet),@echo "    [STRIP] $@ < $<")
-	$(qexec)$(STRIP) --strip-unneeded \
-         `$(NM) $< | grep ' [A-TV-Z] ' | awk '{print "-K"$$3'}`\
-          -o $@ $<
+	$(qexec)cp $< $@
+	$(qexec)$(STRIP) --strip-unneeded $@
 else
 %.a: %_g.a
 	$(if $(quiet),@echo "    [CP] $@ < $<")
